### PR TITLE
Updating known issue for search-collector 2.4

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/17208

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -117,7 +117,11 @@ For ROKS clusters, {product-title-short} observability does not display data in
 === High CPU usage by the search-collector pod
 //2.3.13897
 
-When search is disabled on a hub cluster that manages 1000 clusters, the `search-collector` pod CPU usage levels are higher than normal. For a four-day period, usage was approximately 2.148Mi of CPU. You can reduce memory usage by reducing the `search-collector` to `0` replicas.
+When search is disabled on a hub cluster that manages 1000 clusters, the `search-collector` pod crashes due to the out-of-memory error (OOM). Complete the following steps:
+
+. If search is disabled on the hub cluster, which means the `search-redisgraph-pod` is not deployed, reduce memory usage by reducing the `search-collector` to `0` replicas.
+
+. If search is enabled on the hub cluster, which means the `search-redisgraph-pod` is deployed, increase the memory usage by editing the `search-collector` deployment.
```
Comment (SherinV) [inline release_notes/known_issues.adoc]: If search is enabled on the hub cluster, which means the `search-redisgraph-pod` is deployed, increase the memory allocated by editing the memory limits on the `search-collector` deployment.

**Review by SherinV — CHANGES_REQUESTED** [review]

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: @SherinV I updated the sentence to read the following: If search is enabled on the hub cluster, which means the `search-redisgraph-pod` is deployed, increase the allocated memory by editing the `search-collector` deployment.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -117,7 +117,11 @@ For ROKS clusters, {product-title-short} observability does not display data in
 === High CPU usage by the search-collector pod
 //2.3.13897
 
-When search is disabled on a hub cluster that manages 1000 clusters, the `search-collector` pod CPU usage levels are higher than normal. For a four-day period, usage was approximately 2.148Mi of CPU. You can reduce memory usage by reducing the `search-collector` to `0` replicas.
+When search is disabled on a hub cluster that manages 1000 clusters, the `search-collector` pod crashes due to the out-of-memory error (OOM). Complete the following steps:
+
+. If search is disabled on the hub cluster, which means the `search-redisgraph-pod` is not deployed, reduce memory usage by reducing the `search-collector` to `0` replicas.
+
```
Comment (SherinV) [inline release_notes/known_issues.adoc:123]: reduce memory usage by scaling down the `search-collector` deployment to `0` replicas.

Comment (dockerymick) [reply release_notes/known_issues.adoc:123]: Updated

**Review by SherinV — APPROVED** [review]

**Review by SherinV — APPROVED** [review]

**Pull request merged**